src/main/java/com/nike/cerberus/domain/cloudformation/DatabaseOutputs.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2017 Nike, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nike.cerberus.domain.cloudformation;
+
+/**
+ * Represents the vpc stack outputs.
```

Comment:
Is port included below?  I'm guessing so in the connection String.

(sorry if I'm not understanding, the parameter just seemed off earlier)

Reply:
@tlisonbee Yes, the port is included in the connection string.

The database stack gets the port form the security-groups stack since the DB is reliant on the SG stack for it's security group